quasar/main.tf

```diff
@@ -118,3 +118,8 @@ module "storage" {
   versioning = true
   private    = true
 }
+
+module "quasar_s3_export_role" {
```

Comment:
Based on the name of this module, I wonder if it should _just_ have the role in it?

We could potentially use our `s3_bucket` module to create the Glacier bucket and set up archive rules:

```hcl
module "quasar_archive" {
  source = "../components/s3_bucket"

  application = "dosomething-quasar"
  name        = "dosomething-quasar-archive"
  environment = "production"
  stack       = "data"

  versioning = true
  archived   = true
  private    = true
}

module "rds_export_role" {
  source = "../components/quasar_s3_export_role"
  
  arn = module.quasar_archive.arn
}
```